test/src/streams/streamsTest.cpp

```diff
@@ -0,0 +1,68 @@
+#include <RGLFields.hpp>
+#include <gtest/gtest.h>
+#include <utils.hpp>
+#include "testKernel.hpp"
+
+using namespace testing;
+
+class StreamsTest : public RGLAutoCleanupTest {
+protected:
+};
+
+TEST_F(StreamsTest, basic_streams)
+{
+    // Prepare two non-default streams.
+    cudaStream_t graphStream;
+    cudaStream_t copyStream;
+
+    cudaStreamCreate(&graphStream);
+    cudaStreamCreate(&copyStream);
+
+    int N = 20000;
+    size_t size = N * sizeof(float);
+
+    // Allocate input vector h_A in host memory
+    float* h_A = (float*)malloc(size);
+
+    // Initialize input vectors
+    for (int i = 0; i < N; ++i) {
+        h_A[i] = i;
+    }
+    // Create event for sensitive job (kernel) end.
+    cudaEvent_t jobDoneEvent;
+    // Create event without timer (used for profiling).
+    // This way, we have got some speed, and also according to CUDA documentation, this flag is neccessary to make sure that multi-stream synchronization is bulltetproof.
```

Comment:
Can you link the doc regarding the "bulletproofness" of not-timing events? Does timing event fail to synchronize? This is important info.